### PR TITLE
Removed font tags + improve markdown /no-checked-out-data

### PR DESCRIPTION
**Pull request by tiagov8**

### CODE REVIEW

File: rules/do-you-confirm-there-is-no-checked-out-data/rule.md [hunk 1]
```diff
@@ -54,75 +51,49 @@ Fill the other fields like below:
...
-
 
+You currently have the following pages checked out:
 
-> • 
-            <font color="#3a66cc"><a href="/rules/do-you-confirm-there-is-no-checked-out-data">http://&lt;siteurl&gt;/DesignandPresentation/RulesToBetterVideoRecording/Pages/Default.aspx</a>  (parent folder)</font>
-> • 
-            <font color="#3a66cc"><a href="/rules/do-you-confirm-there-is-no-checked-out-data">http://&lt;siteurl&gt;/DesignandPresentation/RulesToBetterVideoRecording/Pages/testing-rule.aspx</a>  (parent folder)</font>
+- http://&lt;siteurl&gt;/DesignandPresentation/RulesToBetterVideoRecording/Pages/Default.aspx  (parent folder)</font>
+- http://&lt;siteurl&gt;/DesignandPresentation/RulesToBetterVideoRecording/Pages/testing-rule.aspx  (parent folder)</font>
```
Comment (JackDevAU) [inline rules/do-you-confirm-there-is-no-checked-out-data/rule.md:88]: What is the correct url for these?

File: rules/do-you-confirm-there-is-no-checked-out-data/rule.md [hunk 2]
```diff
@@ -54,75 +51,49 @@ Fill the other fields like below:
...
 
-
-
-Tip: See all files you have checked out at [<font color="#3a66cc">http://&lt;siteurl&gt;/_layouts/<span>SSWReports/CheckedOutReport.aspx</span></font>](/do-you-confirm-there-is-no-checked-out-data)
+**Tip:** See all files you have checked out at [http://&lt;siteurl&gt;/_layouts/SSWReports/CheckedOutReport.aspx](/do-you-confirm-there-is-no-checked-out-data)
```
Comment (JackDevAU) [inline rules/do-you-confirm-there-is-no-checked-out-data/rule.md:90]: ^ same as above comment

File: rules/do-you-confirm-there-is-no-checked-out-data/rule.md [hunk 3]
```diff
@@ -54,75 +51,49 @@ Fill the other fields like below:
...
- 
-
 You have some pages checked out in SharePoint.
 
 
-> 1. You should check in at least daily. Revise our SSW rule [<font color="#3a66cc">on Frequent SharePoint Check-ins</font>](/do-you-confirm-there-is-no-checked-out-data).
-> 2. If you are no longer editing these files, check them in! 
-> 
-> 3. Reply to this email with something like:
+1. You should check in at least daily. Revise our SSW rule [<font color="#3a66cc">on Frequent SharePoint Check-ins</font>](/do-you-confirm-there-is-no-checked-out-data).
```
Comment (JackDevAU) [inline rules/do-you-confirm-there-is-no-checked-out-data/rule.md:80]: Should we keep the font color on this?

**Review by JackDevAU — CHANGES_REQUESTED** [review]

File: rules/do-you-confirm-there-is-no-checked-out-data/rule.md (hunk 1 above)
Comment (tiagov8) [reply rules/do-you-confirm-there-is-no-checked-out-data/rule.md]: These are example URLs. I changed to use the placeholder ;)

File: rules/do-you-confirm-there-is-no-checked-out-data/rule.md (hunk 3 above)
Comment (tiagov8) [reply rules/do-you-confirm-there-is-no-checked-out-data/rule.md]: No! Fixed

**Review by JackDevAU — APPROVED** [review]

**Pull request merged**